### PR TITLE
index updated

### PR DESCRIPTION
**Pull request by SidharthSasikumar**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,16 +1,12 @@
...
 <head>
-	<meta charset="utf-8" />
-	<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
+	<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" charset="utf-8"  />
 
     <meta content='width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0' name='viewport' />
     <meta name="viewport" content="width=device-width" />
 
 
-    <meta name="description" content="Adly Loukil is an English Teacher specilizing in providing professional student tailored lessons of the highest quality. To book a lesson with him please contact him via your desired form of communication. Line and Email is most prefered. All levels and abilities are welcome!">
```
Comment (dennerkrans) [inline index.html]: Why did you remove these?

**Comment by adlyloukil** [comment]
@SidharthSasikumar if your going to contribute please contribute productively thanks. Also thank you @dennerkrans for reviewing the pull request.
